src/client/power-pages/actions-hub/ActionsHubTreeDataProvider.ts

```diff
@@ -61,7 +71,29 @@ export class ActionsHubTreeDataProvider implements vscode.TreeDataProvider<Actio
         }
     }
 
+    public refresh(): void {
+        this._onDidChangeTreeData.fire();
+    }
+
     public dispose(): void {
         this._disposables.forEach(d => d.dispose());
     }
+
+    private registerPanel(pacTerminal: PacTerminal): vscode.Disposable[] {
+        const pacWrapper = pacTerminal.getWrapper();
+        return [
+            vscode.commands.registerCommand("powerpages.actionsHub.refresh", async () => {
+                try {
+                    const pacActiveAuth = await pacWrapper.activeAuth();
+                    if (pacActiveAuth && pacActiveAuth.Status === SUCCESS) {
+                        const authInfo = extractAuthInfo(pacActiveAuth.Results);
+                        pacAuthManager.setAuthInfo(authInfo);
+                    }
+                    this.refresh();
```

Comment:
Do we need to explicitly call `refresh` here? As I understand, when we call `setAuthInfo`, it will call `onDidChangeEnvironment` which is wired to call `this.refresh`.

Reply:
Good pick, we can remove it.